truss-chains/truss_chains/public_api.py

```diff
@@ -223,7 +225,7 @@ class HelloWorld(chains.ChainletBase):
             with chains.run_local(
                 secrets={"some_token": os.environ["SOME_TOKEN"]},
                 chainlet_to_service={
-                    "SomeChainlet": chains.ServiceDescriptor(
+                    "SomeChainlet": chains.DeployedServiceDescriptor(
```

Comment:
I assume you did a full text search+replace on all the symbols that were changed/renamed in this PR. - so this is updated in all other potential places too.